charts/retool/templates/deployment_multiplayer_ws.yaml

```diff
@@ -3,14 +3,37 @@ apiVersion: v1
 kind: Service
 metadata:
   name: {{ template "retool.multiplayer.name" . }}
+  labels:
+    {{- include "retool.labels" . | nindent 4 }}
+  {{- if .Values.service.labels }}
+  {{- range $key, $value := .Values.service.labels }}
+    {{ $key }}: {{ $value | quote }}
+  {{- end }}
+  {{- end }}
+  {{- if .Values.multiplayer.service.annotations }}
+  {{- with .Values.multiplayer.service.annotations }}
+  annotations:
+    {{- range $key, $value := . }}
+      {{ $key }}: {{ $value | quote }}
+    {{- end }}
+  {{- end }}
+  {{- end }}
 spec:
   selector:
     retoolService: {{ template "retool.multiplayer.name" . }}
   ports:
   - name: http-server
     protocol: TCP
+    {{- if .Values.multiplayer.service.externalPort }}
```

Comment:
There's no need for an if/else here since you defined a default value for externalPort in values.yaml! Feel free to just have the `port: {{ .Values.service.externalPort }}` here.

Reply:
With this if/else, it's backwards compatible in case people don't have the `externalPort` in their current `values.yaml`. If we don't have it, then if people quietly upgrade their chart without adding to `values.yaml`, things would likely break.

Reply:
Ah, thanks for the explanation! 

Apologies for being unclear, since you set a default value in values.yaml, it should carry over if they did not define it. I thought our default values.yaml would still act as the final fallback set of values in the event that someone didn't have a value set, but I'm hazy now and don't remember for sure.

Since you're leaving this check, I did notice that you've used `.Values.multiplayer.service.externalPort` as the if condition, but `.Values.service.externalPort` as the actual value being set. Should they match?

Reply:
Oh yeah that was a mistake, it should've all referred to the `multiplayer` part of the config. Thanks for fixing it directly.